### PR TITLE
Re-introduce platform-browser-dynamic 

### PR DESCRIPTION
**Pull request by vicb**
To avoid breaking user apps (deprecated)

### CODE REVIEW

File: modules/@angular/platform-browser-dynamic/testing.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * from '@angular/platform-browser/testing';
+
```
Comment (IgorMinar) [inline modules/@angular/platform-browser-dynamic/testing.ts]: add console.log please

File: modules/@angular/platform-browser-dynamic/testing_e2e.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from '@angular/platform-browser/testing/e2e_util';
```
Comment (IgorMinar) [inline modules/@angular/platform-browser-dynamic/testing_e2e.ts]: add console.log please

**Comment by IgorMinar** [comment]
the rest looks good

**Comment by vicb** [comment]
merged via 172a5663ef548962aa4efa96efacbff08fd67b91

**Pull request closed without merging**